help/follow-a-topic.md

```diff
@@ -0,0 +1,32 @@
+# Follow a topic
+
+Zulip lets you follow topics you are interested in, and configure what
+notifications you'd like to receive for topics you follow.
+
+In muted streams, topics you follow are also automatically treated as
+[unmuted](/help/mute-a-topic).
+
+## Configure topic notifications
```

Comment:
Maybe this subheading should explicitly say that these instructions allow users to start or stop following a topic?

```suggestion
## Follow or unfollow a topic
```